Mathlib/CategoryTheory/Abelian/Basic.lean

```diff
@@ -498,6 +499,44 @@ theorem monoLift_comp [Mono f] {T : C} (g : T ⟶ Y) (hg : g ≫ cokernel.π f =
     WalkingParallelPair.zero
 #align category_theory.abelian.mono_lift_comp CategoryTheory.Abelian.monoLift_comp
 
+section
+
+variable {D : Type*} [Category D] [HasZeroMorphisms D]
+
+/-- If `F : D ⥤ C` is a functor to an abelian category, `i : X ⟶ Y` is a morphism
+admitting a cokernel such that `F` preserves this cokernel and `F.map i` is a mono,
+then `F.map X` identifies to the kernel of `F.map (cokernel.π i)`. -/
+noncomputable def isLimitMapConeOfKernelForkOfιCokernelConditionOfMono
```

Comment:
These names are intimidatingly long... would `isLimitMapConeOfKernelForkOfι` be specific enough?

Reply:
Thanks for the suggestions! I have shortened these names.